indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/utils/PoolUtils.kt

```diff
@@ -1,37 +1,25 @@
 package com.luxoft.blockchainlab.hyperledger.indy.utils
 
+import org.apache.commons.io.FileUtils
 import org.hyperledger.indy.sdk.IndyException
```

Comment:
Why did you change that file?
E.g "io.FileUtils" conflicts with Corda dependencies in supply-chain project.

Could you revert all files not related to implementation? We can check and merge them in separated PR.
It also the same regarding unnecessary formatting. It shouldn't be in the same PR as a new feature.

 Please, revert all not related changes back (formatting, files, etc)